src/drivers/ist8310/ist8310.cpp

```diff
@@ -1279,6 +1280,9 @@ struct ist8310_bus_option {
 #ifdef PX4_I2C_BUS_ONBOARD
 	{ IST8310_BUS_I2C_INTERNAL, "/dev/ist8310_int", PX4_I2C_BUS_ONBOARD, NULL },
 #endif
+#ifdef PX4_I2C_BUS_EXPANSION2
+	{ IST8310_BUS_I2C_EXTERNAL2, "/dev/inst8310_ext", PX4_I2C_BUS_EXPANSION2, NULL },
```

Comment:
typo: should be ist8310_ext... what if you have both connected? It should probably be ist8310_ext2 otherwise would conflict.